docs/specs/tripgo.swagger.yaml

```diff
@@ -540,6 +540,23 @@ paths:
           in: query
           description: Boolean if routing result should include only the best trip found for query.
           default: false
+        - name: cyclingAlt
+          type: string
+          enum:
+            - best
+            - shortest
+            - friendliest
+            - all
+            - allButShortest
```

Comment:
```suggestion
```

Let’s not document that “all but shortest” publicly as it’s a temporary solution until this is turned into an array.